cpp/ps0004/problem solve 4.cpp

```diff
@@ -2,22 +2,27 @@
 using namespace std;
 int main()
 {
+    // prompting user for number 1.5/1.5
     float num;
     cout << "Enter a number from 32 to 126:\n";
     cin >> num;
-    if ((num >= 32) && (num <=126)) {
-    char asciiChar = (char) num;
-    cout << "The character for " << num << " is: " << asciiChar ;
+
+    // checking invalidaty of user's input 4/4
+    // print the character 2/2
+    if (num!=(int)num) {
+        cout << "Invalid a floating point number\n";
     }
-    else if (num!=(int)num) {
-    cout << "Invalid a floating point number\n";
+    else if ((num >= 32) && (num <=126)) {
+        char asciiChar = (char) num;
+        cout << "The character for " << num << " is: " << asciiChar ;
     }
     else if (num==0) {
-    cout << "Invalid character/s or zero value\n";
+        cout << "Invalid character/s or zero value\n";
     }
     else {
-    cout << "Invalid number is out of the range\n";
+        cout << "Invalid number is out of the range\n";
     }
```

Comment:
- [x] used `if` structure in a right way
- [x] `if` structure take in consideration all invalid cases
- [x] all expressions to check invalidity are correct
- [x] used `std::cout` and `std::endl` or `\n` to print a different message for the 3 cases
- [x] converted the number to the opposite printable character
- [x] printed the number and the opposite printable character